### PR TITLE
Fix 0.6.0 bugs.

### PR DESCRIPTION
**Pull request by NodeGuy**
> Thank you a lot for contributing to Cosmos Voyager!

### Issue

closes: #720 

### Screenshots

❤️ Thank you!

### CODE REVIEW

File: app/src/renderer/vuex/modules/node.js [hunk 1]
```diff
@@ -145,14 +145,7 @@ export default function({ node }) {
...
       node.rpcReconnect()
-
-      if (mocked) {
-        // if we run a mocked version only, we don't want the lcd to run in the meantime
-        ipcRenderer.send("stop-lcd")
-
-        // the mocked node is automatically connected
-        dispatch("reconnected")
-      }
+      dispatch("reconnected")
```
Comment (NodeGuy) [inline app/src/renderer/vuex/modules/node.js]: @faboweb Please review this change; it fixed the bug but it's unclear to me why we originally wanted to stop the LCD.

File: app/src/renderer/components/common/AppMenu.vue [hunk 2]
```diff
@@ -3,7 +3,10 @@ menu.app-menu
   .app-menu-main
     list-item(to="/" exact @click.native="close" title="Wallet")
     list-item(to="/wallet/transactions" exact @click.native="close" title="Transactions" v-if="config.devMode")
-    list-item(to="/staking" exact @click.native="close" title="Staking" v-bind:class="{ 'active': isValidatorPage }" v-if="config.devMode")
+
+    // Hide staking for now (see https://github.com/cosmos/voyager/issues/720).
+    //- list-item(to="/staking" exact @click.native="close" title="Staking" v-bind:class="{ 'active': isValidatorPage }" v-if="config.devMode")
```
Comment (faboweb) [inline app/src/renderer/components/common/AppMenu.vue]: we have a development mode flag `v-if="config.devMode"` that is already applied. If the staking option is still visible, the user has `NODE_ENV=development` set globally.

File: app/src/renderer/vuex/modules/node.js (hunk 1 above)
Comment (faboweb) [reply app/src/renderer/vuex/modules/node.js]: Do you let your car running while you use your bike? ;)

**Comment by faboweb** [comment]
I fixed the reconnection bug in #726 while shutting down the lcd process. If this is acceptable let's use that fix.

**Pull request closed without merging**